tools/getrealdeps.rb

```diff
@@ -1,30 +1,39 @@
 #!/usr/bin/env ruby
-# getrealdeps for Chromebrew version 1.2
+# getrealdeps for Chromebrew version 1.3
 # Author: Satadru Pramanik (satmandu) satadru at gmail dot com
 require 'fileutils'
 
+if ARGV.include?('--use-crew-dest-dir')
+  ARGV.delete('--use-crew-dest-dir')
+  @opt_use_crew_dest_dir = true
+end
+
 # Exit quickly if an invalid package name is given.
 if ARGV[0].nil? || ARGV[0].empty? || ARGV[0].include?('#')
-  puts 'Getrealdeps checks for the dependencies of a package.'
-  puts 'Dependencies are added if the package file is missing them.'
-  puts 'Usage: getrealdeps.rb <packagename>'
+  puts 'Getrealdeps checks for the runtime dependencies of a package.'
+  puts 'The runtime dependencies are added if the package file is missing them.'
+  puts 'Usage: getrealdeps.rb [--use_crew_dest_dir] <packagename>'
   exit 1
 end
 
 pkg = ARGV[0].chomp('.rb')
+puts "Checking for the runtime dependencies of #{pkg}..."
 
 CREW_PREFIX = ENV['CREW_PREFIX'] || `crew const CREW_PREFIX`.split('=')[1].chomp
+CREW_DEST_DIR = ENV['CREW_DEST_DIR'] || `crew const CREW_DEST_DIR`.split('=')[1].chomp
 CREW_LIB_PREFIX = ENV['CREW_LIB_PREFIX'] || `crew const CREW_LIB_PREFIX`.split('=')[1].chomp
```

Comment:
Doesn't pass the 3 or more test to justify `require 'lib/const'`?

Reply:
We should make it a greater than 3 rule... because `require 'lib/const'` needs the path to be right, and we might be anywhere in a cloned git tree.

Reply:
Doesn't `CREW_LOCAL_REPO_ROOT` account for that?
```
require CREW_LOCAL_REPO_ROOT.to_s + '/lib/const'
```
?

Reply:
Actually, now that I thought about it a bit, `CREW_LOCAL_REPO_ROOT` is loaded from const.rb. :blush:

Reply:
Done.